### ISSUE TITLE
Can't set a description correctly

### ISSUE THREAD

**Issue by zackehh** [tlivings/enjoi#23]
My bad if I'm missing something, but I used code similar to the example and my description is not being set:

```javascript
Enjoi({
  type: 'integer',
  minimum: 0,
  default: 1,
  description: 'a query param',
})
```

The above gives me something like this when logged out:

```javascript
{ isJoi: true,
  _type: 'number',
  _settings: null,
  _valids: { _set: [] },
  _invalids: { _set: [ Infinity, -Infinity ] },
  _tests: 
   [ { func: [Function],
       name: 'integer',
       arg: undefined,
       options: undefined },
     { func: [Function], name: 'min', arg: 0, options: undefined } ],
  _refs: [],
  _flags: {},
  _description: null,
  _unit: null,
  _notes: [],
  _tags: [],
  _examples: [],
  _meta: [],
  _inner: {} }
```

But if I use `Joi` directly, I can see the description:

```javascript
Joi.number().integer().min(0).default(1).description('a query param');
```

```javascript
{ isJoi: true,
  _type: 'number',
  _settings: null,
  _valids: Set { _set: [] },
  _invalids: Set { _set: [ Infinity, -Infinity ] },
  _tests: 
   [ { func: [Function],
       name: 'integer',
       arg: undefined,
       options: undefined },
     { func: [Function], name: 'min', arg: 0, options: undefined } ],
  _refs: [],
  _flags: { default: 1 },
  _description: 'a query param',
  _unit: null,
  _notes: [],
  _tags: [],
  _examples: [],
  _meta: [],
  _inner: {} }
```

Am I perhaps doing something wrong, or might this be a bug?

Thank you!

**Comment by tlivings** [comment]
Looks like a bug, checking.

**Comment by tlivings** [comment]
Fixed in `2.1.0`.
